### PR TITLE
[12.0][IMP] connector_jira: worklog/timesheet date timezone

### PR DESCRIPTION
**Pull request by alexey-pelykh**
@guewen @simahawk this PR adds a setting to configure how to treat datetime from worklog.

### CODE REVIEW

**Review by guewen — APPROVED** [review]

File: connector_jira/models/account_analytic_line/importer.py [hunk 1]
```diff
@@ -49,6 +49,21 @@ def issue(self, record):
             refs['jira_epic_issue_key'] = issue['fields'][epic_field_name]
         return refs
 
+    @mapping
+    def date(self, record):
+        mode = self.backend_record.worklog_date_timezone_mode
+        started = record['started']
+        _logger.info('!!!! %s %s' % (mode, started))
```
Comment (simahawk) [inline connector_jira/models/account_analytic_line/importer.py]: is it needed to log here? especially as INFO?

File: connector_jira/tests/test_import_analytic_line.py [hunk 2]
```diff
@@ -108,3 +108,85 @@ def _test_import_worklog(self, expected_project, expected_task):
                 'user_id': self.env.user.id,
             }]
         )
+
```
Comment (simahawk) [inline connector_jira/tests/test_import_analytic_line.py:111]: am I wrong or you are not testing if the correct TZ is applied?

**Review by simahawk — CHANGES_REQUESTED** [review]

File: connector_jira/models/account_analytic_line/importer.py (hunk 1 above)
Comment (alexey-pelykh) [reply connector_jira/models/account_analytic_line/importer.py]: Sorry, leftover, removed

File: connector_jira/tests/test_import_analytic_line.py (hunk 2 above)
Comment (alexey-pelykh) [reply connector_jira/tests/test_import_analytic_line.py:111]: I do, there are 3 more testcases

Comment (alexey-pelykh) [reply connector_jira/tests/test_import_analytic_line.py:111]: `test_import_worklog_specific`
`test_import_worklog_user`
and
`test_import_worklog_naive`

Comment (alexey-pelykh) [reply connector_jira/tests/test_import_analytic_line.py:111]: Only `specific` and `user` actually shifts the date

Comment (simahawk) [reply connector_jira/tests/test_import_analytic_line.py:111]: oh yeah, sorry, did not catch the date changing.
TIP: next time add a comment on top of it saying what you expect ;)

File: connector_jira/__manifest__.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'JIRA Connector',
-    'version': '12.0.1.2.0',
+    'version': '12.0.1.3.0',
```
Comment (simahawk) [inline connector_jira/__manifest__.py:5]: FTR this is not needed as we use the bot to merge ;)

**Review by simahawk — APPROVED** [review]

File: connector_jira/__manifest__.py (hunk 3 above)
Comment (alexey-pelykh) [reply connector_jira/__manifest__.py:5]: :) Old habit :)

**Comment by alexey-pelykh** [comment]
Let's merge if possible? I know we need 3 approves, but I guess there's not much active users

**Pull request merged**